scripts/lease-period-prolongator/README.md

```diff
@@ -0,0 +1,36 @@
+# lease-period-prolongator
+
```

Comment:
Nit: A short description of the functionality would be appreciated.

Reply:
Added.